### ISSUE TITLE
Using index_generator.yaml fails

### ISSUE THREAD

**Issue by nicolasmol** [BreX900/index_generator#7]
Hi
It works only when adding the instruction in pubspec.yaml.
If I add the instructions only in a separated index_generator.yaml it returns: 
```[39m[31mline 1, column 1 of pubspec.yaml: Missing key "index_generator". type 'Null' is not a subtype of type 'Map<dynamic, dynamic>' in type cast```

```
index_generator:
  default_name: '_exports'
  exclude:
    - '**.g.dart'
    - '**_state.dart'
    - '**_event.dart'
    - 'imports.dart'
    - '_exports.dart'
  indexes: 
    - path: 'lib/presentation'
      name: '_exports'
    - path: 'lib/core'
      name: '_exports'
    - path: 'lib/logic'
      name: '_exports'
    - path: 'lib/data_providers'
      name: '_exports'
```

**Comment by BreX900** [comment]
@nicolasmol Yes there is a bug, I will fix it in the day, in time you can use `dart pub global run index_generator -s index_generator.yaml`
Use `--help` to see available commands

Thanks a lot for reporting

**Comment by BreX900** [comment]
@nicolasmol Could you try and close the issue in case everything works fine in the latest version? `3.3.2`

**Comment by nicolasmol** [comment]
I confirm it's fixed in `3.3.2`.
Thanks for your promptness
